integration-test/src/test/java/org/apache/iotdb/db/it/IoTDBConfigNodeIT.java

```diff
@@ -204,6 +228,597 @@ public void removeAndStopConfigNodeTest() {
+    try (SyncConfigNodeIServiceClient client =
+        (SyncConfigNodeIServiceClient) EnvFactory.getEnv().getConfigNodeConnection()) {
+      List<ConfigNodeWrapper> configNodeWrappers = EnvFactory.getEnv().getConfigNodeWrapperList();
+      List<DataNodeWrapper> dataNodeWrappers = EnvFactory.getEnv().getDataNodeWrapperList();
+
+      // test showCluster
+      TShowClusterResp clusterNodes = getClusterNodeInfos(client, 1, 3);
+      List<TConfigNodeLocation> configNodeLocations = clusterNodes.getConfigNodeList();
+      List<TDataNodeLocation> dataNodeLocations = clusterNodes.getDataNodeList();
+
+      for (TConfigNodeLocation configNodeLocation : configNodeLocations) {
+        boolean found = false;
+        for (ConfigNodeWrapper configNodeWrapper : configNodeWrappers) {
+          if (configNodeWrapper.getIp().equals(configNodeLocation.getInternalEndPoint().getIp())
+              && configNodeWrapper.getPort() == configNodeLocation.getInternalEndPoint().getPort()
+              && configNodeWrapper.getConsensusPort()
+                  == configNodeLocation.getConsensusEndPoint().getPort()) {
+            found = true;
+            break;
+          }
+        }
+        assertTrue(found);
+      }
+
+      for (TDataNodeLocation dataNodeLocation : dataNodeLocations) {
+        boolean found = false;
+        for (DataNodeWrapper dataNodeWrapper : dataNodeWrappers) {
+          if (dataNodeWrapper.getIp().equals(dataNodeLocation.getInternalEndPoint().getIp())
+              && dataNodeWrapper.getPort() == dataNodeLocation.getClientRpcEndPoint().getPort()
+              && dataNodeWrapper.getInternalPort()
+                  == dataNodeLocation.getInternalEndPoint().getPort()
+              && dataNodeWrapper.getMppDataExchangePort()
+                  == dataNodeLocation.getMPPDataExchangeEndPoint().getPort()
+              && dataNodeWrapper.getSchemaRegionConsensusPort()
+                  == dataNodeLocation.getSchemaRegionConsensusEndPoint().getPort()
+              && dataNodeWrapper.getDataRegionConsensusPort()
+                  == dataNodeLocation.getDataRegionConsensusEndPoint().getPort()) {
+            found = true;
+            break;
+          }
+        }
+        assertTrue(found);
+      }
+
+      // test showDataNodes
+      TShowDataNodesResp showDataNodesResp = client.showDataNodes();
+      List<TDataNodeInfo> dataNodesInfo = showDataNodesResp.getDataNodesInfoList();
+
+      for (TDataNodeInfo dataNodeInfo : dataNodesInfo) {
+        boolean found = false;
+        for (DataNodeWrapper dataNodeWrapper : dataNodeWrappers) {
+          if (dataNodeWrapper.getIp().equals(dataNodeInfo.getRpcAddresss())
+              && dataNodeWrapper.getPort() == dataNodeInfo.getRpcPort()) {
+            found = true;
+            break;
+          }
+        }
+        assertTrue(found);
+      }
+    } catch (Exception e) {
+      e.printStackTrace();
+      fail(e.getMessage());
+    }
```

Comment:
Just throw any Exceptions in the interface, we don't need to catch and print them since it's a test code